androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -285,15 +336,16 @@ fun EnhancedVideoPlayer(
                     brightnessMutableInteractionSource = brightnessMutableInteractionSource,
                     volumeMutableInteractionSource = volumeMutableInteractionSource,
                     timeBarMutableInteractionSource = timeBarMutableInteractionSource,
-                    totalDuration = totalDuration,
+                    // hard coded duration so it doesnt crash, working on a fix
+                    totalDuration = 634566,
```

Comment:
Just a question, why when you hard coded it works?

Reply:
sometimes the totalDuration would be a negative value and it would crash the app, the hard coded number ensures it is positive